### PR TITLE
#1416 created ReceiveActor implementation of AtLeastOnceDeliveryActor

### PR DESCRIPTION
**Pull request by maxim-s**

### CODE REVIEW

**Comment by Horusiath** [comment]
:+1: Well done :) Build is failing but this seems to be more related to multinode test runner than the changes introduced.

**Comment by Aaronontheweb** [comment]
@Horusiath 

```

 SqliteJournalQuerySpec.Journal_queried_on_Manifest_returns_events_with_particular_manifest	
 
First failure:  	1469	#1618	maxim.salamatk… (1)	30 Nov 15 09:38
Failed: Timeout 00:00:03 while waiting for a message of type Akka.Persistence.WriteMessagesSuccessful 
Expected: True
Actual:   False
   at Akka.TestKit.Xunit2.XunitAssertions.Fail(String format, Object[] args) in D:\BuildAgent\work\49b164d63843fb4\src\contrib\testkits\Akka.TestKit.Xunit2\XunitAssertions.cs:line 22
   at Akka.TestKit.TestKitBase.InternalExpectMsgEnvelope[T](Nullable`1 timeout, Action`2 assert, String hint, Boolean shouldLog) in D:\BuildAgent\work\49b164d63843fb4\src\core\Akka.TestKit\TestKitBase_Expect.cs:line 196
   at Akka.TestKit.TestKitBase.InternalExpectMsgEnvelope[T](Nullable`1 timeout, Action`1 msgAssert, Action`1 senderAssert, String hint) in D:\BuildAgent\work\49b164d63843fb4\src\core\Akka.TestKit\TestKitBase_Expect.cs:line 179
   at Akka.TestKit.TestKitBase.InternalExpectMsg[T](Nullable`1 timeout, Action`1 msgAssert, String hint) in D:\BuildAgent\work\49b164d63843fb4\src\core\Akka.TestKit\TestKitBase_Expect.cs:line 154
   at Akka.TestKit.TestKitBase.ExpectMsg[T](Nullable`1 duration, String hint) in D:\BuildAgent\work\49b164d63843fb4\src\core\Akka.TestKit\TestKitBase_Expect.cs:line 29
   at Akka.Persistence.Sql.Common.TestKit.SqlJournalQuerySpec.WriteEvents() in D:\BuildAgent\work\49b164d63843fb4\src\contrib\persistence\Akka.Persistence.Sql.Common.TestKit\SqlJournalQuerySpec.cs:line 123
   at Akka.Persistence.Sqlite.Tests.SqliteJournalQuerySpec..ctor(ITestOutputHelper output) in D:\BuildAgent\work\49b164d63843fb4\src\contrib\persistence\Akka.Persistence.Sqlite.Tests\SqliteJournalQuerySpec.cs:line 23
```

**Comment by Aaronontheweb** [comment]
Re-running the MNTR now

File: src/core/Akka.Persistence/Akka.Persistence.csproj [hunk 1]
```diff
@@ -112,4 +114,118 @@
   </Target>
   -->
   
+  <Choose>
```
Comment (Aaronontheweb) [inline src/core/Akka.Persistence/Akka.Persistence.csproj]: Bah! I missed this when I first looked at the PR and merged it.

We'll need to remove that `<Choose>` block below.

Comment (maxim-s) [inline src/core/Akka.Persistence/Akka.Persistence.csproj]: Sorry, my mistake, I'll check it next time

**Pull request merged**